source/background.ts

```diff
@@ -81,15 +80,15 @@ async function isFirstInstall(suggestedReason: string): Promise<boolean> {
 		&& suggestedReason === 'install'
 
 		// Safari reports "install" even on updates #5412
-		&& !(isSafari() && await hasUsedStorage())
+		&& !(await hasUsedStorage())
 	);
 }
 
 chrome.runtime.onInstalled.addListener(async ({reason}) => {
 	// Only notify on install
 	if (await isFirstInstall(reason)) {
```

Comment:
@yakov116 Note that this needs to be set to `true` to work since we disable in on dev builds